dosagelib/comic.py

```diff
@@ -106,8 +106,8 @@ def save(self, basepath):
         fn = fnbase + self.ext
         # compare with >= since content length could be the compressed size
         if os.path.isfile(fn) and os.path.getsize(fn) >= self.contentLength:
-            self._exist_err(fn)
-            return fn, False
+            out.info(u'Skipping existing file "%s".' % fn)
+            getHandler().comicDownloaded(self, fn)
```

Comment:
This looks buggy to me. Does this work with all the different output handlers? Where is the previous code?